pinot-spi/src/main/java/org/apache/pinot/spi/data/DataSchema.java

```diff
@@ -40,7 +40,10 @@
 
 
 /**
- * The <code>DataSchema</code> class describes the schema of {@link DataTable}.
+ * The <code>DataSchema</code> class describes the columnName and columnType of a row.
+ *
+ * <p>Comparing with {@link Schema} it doesn't contain any {@link FieldSpec} information except for the DataType.
```

Comment:
Schema is used to describe the table; DataSchema is used to describe the query results